pkg/nodeconfig/nodeconfig.go

```diff
@@ -364,7 +388,12 @@ func (nc *nodeConfig) Deconfigure() error {
 	}
 
 	// Cordon and drain the Node before we interact with the instance
-	drainHelper := &drain.Helper{Ctx: context.TODO(), Client: nc.k8sclientset}
+	drainHelper := &drain.Helper{
+		Ctx:    context.TODO(),
+		Client: nc.k8sclientset,
+		ErrOut: &ErrWriter{nc.log},
+		Out:    &OutWriter{nc.log},
+	}
```

Comment:
The other instance of `drainHelper` in `nc.Configure()` should be changed as well.

Since the object initialization is a bit more complicated now, how about creating a new constructor:
```
func (nc *nodeConfig) newDrainHelper() *drain.Helper {
  return &drain.Helper{
		Ctx:    context.TODO(),
		Client: nc.k8sclientset,
		ErrOut: &ErrWriter{nc.log},
		Out:    &OutWriter{nc.log},
	}
}
```

Reply:
Should the `drain.Helper` instance be part of the [nodeConfig struct](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/nodeconfig/nodeconfig.go#L48)? So, we don't create a new instance on each constructor call.